### DISCUSSION TITLE
I cant Do much except move around and slide Is this the actual State of the project right now?

### DISCUSSION THREAD

**Discussion by Nocturnal-Neon** [Voidware-Prohibited/ALSXT#40] [Q&A] [answered]
I can move around(including climbing), slide and punch, no other keys do anything (Weapon related stuff is not working at all), Just wanted to know if this is the current state of the project or if its something on my side. I have reinstalled it twice and followed the instructions (at first I tried to put this into an ALS project that I had already been working on for a while and had added alot more functionality to it, so I thought it was just my project). 

Thanks :)

**Comment by smateja** [comment, marked as answer]
Jap - there a are more keys like The P Key :) and a lot of debuggin stuff - there is a combat component still in dev - and i think there will come a lot of more - stay tune. Development is back....

**Reply by Nocturnal-Neon** [reply]
Thanks, I did find the P key after trying every key ahaha. I just want to merge some of the features in this into my own ALS project thats a bit further along. and after looking through it ALSXT is pretty awesome.
